cli/lint-sass.js

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+
+const chalk = require('chalk')
+const fs = require('fs')
+const glob = require('glob-all')
+const path = require('path')
+const sassLint = require('sass-lint')
+
+/**
+ * List of configuration files to look for
+ * @type {Array.<string>}
+ */
+const CONFIG_FILE_NAMES = [
+  '.sasslint.json'
+]
+
+/**
+ * List of glob directories for where to find SASS files
+ * @type {Array.<string>}
+ */
+const TEMPLATE_FILE_LOCATIONS = [
+  'addon/**/*.sass',
+  'addon/**/*.scss',
+  'app/**/*.sass',
+  'app/**/*.scss',
+  'tests/**/*.sass',
```

Comment:
Do we have an `.sass` files or care about linting them? I thought we agreed on `.scss`?